jsk_kinova_robot/jsk_kinova_description/robots/gen3_lite_gen3_lite_2f.xacro

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<robot name="jsk_gen3_lite_gen3_lite_2f" xmlns:xacro="http://ros.org/wiki/xacro">
+    <!-- Arguments -->
+    <xacro:arg name="arm" default="gen3_lite" />
+    <!-- Those arguments are ignored for now on Gen3 lite because there is only one supported configuration -->
+    <xacro:arg name="dof" default="6" />
+    <xacro:arg name="vision" default="false" />
+    <xacro:arg name="gripper" default="gen3_lite_2f" />
+    <xacro:arg name="sim" default="false" />
+    <xacro:arg name="prefix" default="" />
+
+    <xacro:include filename="$(find kortex_description)/robots/kortex_robot.xacro" />
+    <xacro:load_robot arm="$(arg arm)" gripper="$(arg gripper)" dof="$(arg dof)" vision="false" sim="$(arg sim)"  prefix="$(arg prefix)" />
+
+    <link name="kinova_wrist_camera_link">
+      <inertial>
+        <origin xyz="0.0 0.0 0.0" rpy="0 0 0" />
+        <mass value="0.0000000001" />
+        <inertia ixx="0.0000000001" ixy="0.0000000001" ixz="0.0000000001"
+            iyy="0.0000000001" iyz="0.0000000001" izz="0.0000000001" />
```

Comment:
Are these parameters necessary?

Reply:
these parameters are necessary for gazebo load.
gazebo crashes when no interia parameter is set in a link.

Reply:
I think we can write `<link name="fuga" /> `https://github.com/PR2/pr2_common/blob/37061a9c68d97931f6b6b30583e4299b526bb145/pr2_description/urdf/sensors/kinect_camera.urdf.xacro#L48
 @sktometometo , could you check this?

Reply:
@k-okada OK I will check if that will work with a real robot and gazebo.